cli/src/commands/deploy.ts

```diff
@@ -115,8 +120,15 @@ export default class Deploy extends Command {
     return app.id;
   }
 
+  public log(str: string) {
+    const { flags: { json } } = this.parse(Deploy);
```

Comment:
This is a bit strange, since we are parsing the command line arguments on each run - I think no reason for not using a member `quiet: boolean` and then writing

```
if (this.quiet) return;
super.log(str);
```

Reply:
I thought about this and I agree, not sure how to scope the new member though.

Reply:
fixed